### DISCUSSION TITLE
Technical support for NFS PVC backup & Restore

### DISCUSSION THREAD

**Discussion by shyjustack** [vmware-tanzu/velero#6645] [Community support Q&A]
In our setup, we are running Version 1.11.0. My Kubernetes cluster is on-premise volumes; we have an NFS server and a worker node. While creating the PV, I mentioned the share location and NFS server. Everything is working fine.

Backup is successfully working after I delete the content inside the volume while restoring pod and pvc, but data is not restoring.

**Comment by sseago** [comment]
What method are you trying to use for volume backup? For NFS, there is no snapshot support, so the only way that volume backup will work is if you use the fs backup feature. Assuming your velero installation already enables node agent, the easiest way to ensure that you're using fs backup (with either restic or kopia) is pass the `--default-volumes-to-fs-backup` flag on backup creation.
